scripts/bootstrap-desktop.ps1

```
@@ -0,0 +1,114 @@
+if ($InstallDeps) {
+    Write-Step "Stopping running bridge process (if any) before pip install..."
+    $bridgeExePath = Join-Path $RepoRoot ".venv\Scripts\open-fdd-desktop-bridge.exe"
+    $killed = $false
+    try {
+        foreach ($p in Get-Process -Name "open-fdd-desktop-bridge" -ErrorAction SilentlyContinue) {
+            Stop-Process -Id $p.Id -Force -ErrorAction SilentlyContinue
+            $killed = $true
+        }
+    } catch {
+        # best effort
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)